### ISSUE TITLE
LengthWire/LengthCapture limited for a reason?

### ISSUE THREAD

**Issue by michael-brade** [fmadio/pcap2json#36]
See here: https://github.com/fmadio/pcap2json/blob/9a58a2dcfcd6b45b38eefe7e38229a146a97d83b/flow.c#L1580-L1581

This crashes when I do a live capture on my local network because I get sizes of up to 64374 bytes. So why limit it to 14 bits instead of 16?

**Comment by fmadio** [comment]
Because packets are always less than 14bits on the physical network. your
seeing larger sized packets because segment offloading is enabled on the
NIC.

**Comment by michael-brade** [comment]
I see. So am I not supposed to use this with segment offloading? I changed the assert and have it working... :)

**Comment by fmadio** [comment]
correct, we use the application for running analysis on real wore traffic
